.github/workflows/release.yml

```diff
@@ -30,47 +44,12 @@ jobs:
           cargo build --all --release --locked
           cd target/release
           strip viceroy
```

Comment:
`strip viceroy` may not work on Windows, since the file will be called `viceroy.exe`.